src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxConnectionsPerServer.cs

```diff
@@ -64,6 +64,7 @@ public void Set_ValidValues_Success(int validValue)
         [InlineData(3, 2, true)]
         [InlineData(3, 5, false)]
         [OuterLoop("Uses external servers")]
+        [SkipOnPlatform(TestPlatforms.Browser, "MaxConnectionsPerServer not supported on Browser")]
```

Comment:
Should `SkipOnPlatform` be added for the rest of the methods that use `MaxConnectionsPerServer`? Or are we only adding it to this method because it is part of the OuterLoop?

We should address the other tests in a followup PR if not in this one.

Reply:
The whole class should have it actually and this is a duplicate, thanks for catching it.